### PR TITLE
Updated GitHub Campus Program copy for Doc page

### LINKED ISSUE #5212: Content updated for GitHub Education's Docs Page
### What article on docs.github.com is affected?

**About GitHub Campus Program**

_Moderator's note:_ Updated to point to publicly accessible info.

~~We would like to update the [docs page](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/about-github-campus-program) by including the [information here](https://docs.google.com/document/d/1iH1mM2jTTqggvpnfDpRG5QTMBTQV2UwvBtwjnD8t10Y/edit#)~~

We would like to update the [docs page](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/about-github-campus-program) by making [the changes listed in the comment below](https://github.com/github/docs/issues/5212#issuecomment-824442186). 

### What part(s) of the article would you like to see updated?

We want to include questions and answer that we see asked very often. Our program is moving to a self-serve model.
To help support the self-serve model, we want to link our GitHub Docs page with updated content tp our main GitHub Campus Program page. 

### Additional information

@emilyistoofunky mentioned that  we don't use FAQs as a content type on our docs, so would likely rework the information into a format we use. This is perfectly fine with us 👍

### PR DESCRIPTION
**Pull request by bassa846**
### Why:

Closes https://github.com/github/docs/issues/5212

### Check off the following:

- [ ] I have reviewed my changes in staging (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**).
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by peeraw** [comment]
Sent from Nine<http://www.9folders.com/>
________________________________
From: Ashley Bass ***@***.***>
Sent: วันอังคารที่ 25 พฤษภาคม ค.ศ. 2021 02:02 น.
To: github/docs
Cc: Subscribed
Subject: [github/docs] Updated GitHub Campus Program copy for Doc page (#6735)

Why:

Closes [https://github.com//issues/5212<https://github.com/github/docs/issues/5212>]

What's being changed:
Check off the following:

  *    I have reviewed my changes in staging (look for the deploy-to-heroku link in your pull request, then click View deployment).
  *    For content changes, I have completed the self-review checklist<https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review>.

Writer impact (This section is for GitHub staff members only):

  *    This pull request impacts the contribution experience
     *    I have added the 'writer impact' label
     *    I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/github/docs/pull/6735

Commit Summary

  *   Updated GCP copy

File Changes

  *   M content/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-at-your-educational-institution/about-github-campus-program.md<https://github.com/github/docs/pull/6735/files#diff-7dbad36331cbade26d7bd54321b817d642be6a81c6cefcc3fee0c1418fd0c841> (24)

Patch Links:

  *   https://github.com/github/docs/pull/6735.patch
  *   https://github.com/github/docs/pull/6735.diff

**Comment by janiceilene** [comment]
This PR is looking great @bassa846! Let me know when you're ready for writer review and I'll get it triaged 💖

**Comment by bassa846** [comment]
@janiceilene this is ready for review. 
HUGE shoutout to @jmarlena for helping me earlier 😄

File: content/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-at-your-educational-institution/about-github-campus-program.md [hunk 1]
```diff
@@ -13,12 +13,34 @@ versions:
 {% data variables.product.prodname_campus_program %} is a package of premium {% data variables.product.prodname_dotcom %} access for teaching-focused institutions that grant degrees, diplomas, or certificates. {% data variables.product.prodname_campus_program %} includes:
 
 - No-cost access to {% data variables.product.prodname_ghe_cloud %} and {% data variables.product.prodname_ghe_server %} for all of your technical and academic departments
-- Teacher training to master Git and {% data variables.product.prodname_dotcom %} with our Campus Advisor program
+- 50,000 GitHub Actions minutes and 50GB GitHub Packages storage
```
Comment (jmarlena) [inline content/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-at-your-educational-institution/about-github-campus-program.md:16]: Added product variables and a space between `50` and `GB` for an easier read.

```suggestion
- 50,000 {% data variables.product.prodname_actions %} minutes and 50 GB {% data variables.product.prodname_registry %} storage
```

File: content/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-at-your-educational-institution/about-github-campus-program.md [hunk 2]
```diff
@@ -13,12 +13,34 @@ versions:
...
 
-If you're a student or academic faculty and your school isn't partnered with {% data variables.product.prodname_dotcom %} as a {% data variables.product.prodname_campus_program %} school, then you can still individually apply for discounts to use {% data variables.product.prodname_dotcom %}. For more information, see "[Use {% data variables.product.prodname_dotcom %} for your schoolwork](/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-for-your-schoolwork)" or "[Use {% data variables.product.prodname_dotcom %} in your classroom and research](/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-in-your-classroom-and-research/)."
+If you're a student or academic faculty and your school isn't partnered with {% data variables.product.prodname_dotcom %} as a {% data variables.product.prodname_campus_program %} school, then you can still individually apply for discounts to use {% data variables.product.prodname_dotcom %}. To apply for the Student Developer Pack, [see the application form.](https://education.github.com/pack/join)
```
Comment (jmarlena) [inline content/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-at-your-educational-institution/about-github-campus-program.md:44]:
```suggestion
If you're a student or academic faculty and your school isn't partnered with {% data variables.product.prodname_dotcom %} as a {% data variables.product.prodname_campus_program %} school, then you can still individually apply for discounts to use {% data variables.product.prodname_dotcom %}. To apply for the Student Developer Pack, [see the application form](https://education.github.com/pack/join).
```

File: content/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-at-your-educational-institution/about-github-campus-program.md (hunk 1 above)
Comment (jmarlena) [inline content/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-at-your-educational-institution/about-github-campus-program.md:16]: Added product variables and a space between `50` and `GB` for an easier read.

```suggestion
- 50,000 {% data variables.product.prodname_actions %} minutes and 50 GB {% data variables.product.prodname_registry %} storage
```

File: content/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-at-your-educational-institution/about-github-campus-program.md [hunk 3]
```diff
@@ -13,12 +13,34 @@ versions:
...
+
+- You must offer GitHub to all of your technical and academic departments and your school’s logo will be shared on the GitHub Education website as a GitHub Campus Program Partner.
+
+- New organizations in your enterprise are automatically added to your enterprise account. To add organizations that existed before your school joined the GitHub Campus Program, please contact [GitHub Education Support](https://support.github.com/contact/education). For more information about administrating your enterprise, see "[Setting up and managing your enterprise](/github/setting-up-and-managing-your-enterprise)." New organizations in your enterprise are automatically added to your enterprise account. To add organizations that existed before your school joined the GitHub Campus Program, please contact GitHub Education Support. 
+
+
+To read more about GitHub's privacy practices, see ["Global Privacy Practices"](https://docs.github.com/en/github/site-policy/global-privacy-practices)
```
Comment (jmarlena) [inline content/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-at-your-educational-institution/about-github-campus-program.md:34]: Link fix + product variable
```suggestion
To read more about {% data variables.product.prodname_dotcom %}'s privacy practices, see ["Global Privacy Practices"](/github/site-policy/global-privacy-practices)
```

File: content/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-at-your-educational-institution/about-github-campus-program.md [hunk 4]
```diff
@@ -13,12 +13,34 @@ versions:
...
-Any school that can agree to the [terms of the program](https://education.github.com/schools/terms) is welcome to join.
+To read about how GitHub is used by educators, see [GitHub Education stories.](https://education.github.com/stories)
+
+### {% data variables.product.prodname_campus_program %} terms and conditions
+
+- The license is free for one year and will automatically renew for free every 2 years. You may continue on the free license so long as you continue to operate within the terms of the agreement. Any school that can agree to the [terms of the program](https://education.github.com/schools/terms) is welcome to join.
+
+- Please note that the licenses are for use by the whole school. Internal IT departments, academic research groups, collaborators, students, and other non-academic departments are eligible to use the licenses so long as they are not making a profit from its use. Externally funded research groups that are housed at the university may not use the free licenses.
+
+- You must offer GitHub to all of your technical and academic departments and your school’s logo will be shared on the GitHub Education website as a GitHub Campus Program Partner.
```
Comment (jmarlena) [inline content/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-at-your-educational-institution/about-github-campus-program.md:29]: Adding some product variables

```suggestion
- You must offer {% data variables.product.prodname_dotcom %} to all of your technical and academic departments and your school’s logo will be shared on the GitHub Education website as a {% data variables.product.prodname_campus_program %} Partner.
```

File: content/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-at-your-educational-institution/about-github-campus-program.md [hunk 5]
```diff
@@ -13,12 +13,34 @@ versions:
...
+
+- New organizations in your enterprise are automatically added to your enterprise account. To add organizations that existed before your school joined the GitHub Campus Program, please contact [GitHub Education Support](https://support.github.com/contact/education). For more information about administrating your enterprise, see "[Setting up and managing your enterprise](/github/setting-up-and-managing-your-enterprise)." New organizations in your enterprise are automatically added to your enterprise account. To add organizations that existed before your school joined the GitHub Campus Program, please contact GitHub Education Support.
```
Comment (jmarlena) [inline content/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-at-your-educational-institution/about-github-campus-program.md:31]:
```suggestion
- New organizations in your enterprise are automatically added to your enterprise account. To add organizations that existed before your school joined the {% data variables.product.prodname_campus_program %}, please contact [GitHub Education Support](https://support.github.com/contact/education). For more information about administrating your enterprise, see "[Setting up and managing your enterprise](/github/setting-up-and-managing-your-enterprise)." New organizations in your enterprise are automatically added to your enterprise account. To add organizations that existed before your school joined the {% data variables.product.prodname_campus_program %}, please contact GitHub Education Support. 
```

File: content/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-at-your-educational-institution/about-github-campus-program.md [hunk 6]
```diff
@@ -13,12 +13,34 @@ versions:
...
+
+
+To read more about GitHub's privacy practices, see ["Global Privacy Practices"](https://docs.github.com/en/github/site-policy/global-privacy-practices)
+
+### {% data variables.product.prodname_campus_program %} Application Eligibility
+
+- Often times, a campus CTO/CIO, Dean, Department Chair, or Technology Officer signs the terms of the program on behalf of the campus.
+
+- If your school does not issue email addresses, GitHub will reach out to your account administrators with an alternative option to allow you to distribute the student developer pack to your students.
```
Comment (jmarlena) [inline content/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-at-your-educational-institution/about-github-campus-program.md:40]: Product variable
```suggestion
- If your school does not issue email addresses, {% data variables.product.prodname_dotcom %} will reach out to your account administrators with an alternative option to allow you to distribute the student developer pack to your students.
```

**Review by jmarlena — DISMISSED** [review]
These changes look great! 

I just made a few suggested comments that I can apply here so this PR can ship today. 🚢

**Review by jmarlena — APPROVED** [review]
All tests are green! This PR looks ready to merge. ⚡

**Pull request merged**